reactive/vpe_router.py

```diff
@@ -35,8 +35,10 @@ def validate_config():
     except Exception as e:
         remove_state('vpe.configured')
         set_state('blocked', 'validation failed: %s' % e)
-    else:
```

Comment:
Not a nice catch. There should be a finally and else. Else means no exception raised. Finally means everything after including exception. With this change vpe.configured will be set always